.github/workflows/tb-sync.yaml

```diff
@@ -35,46 +41,94 @@ jobs:
         with:
           token: ${{ secrets.TEMPLATES_GITHUB_TOKEN }}
           repository: platformsh/template-builder
-          path: template-builder
+          path: ${{ env.TEMPLATEBUILDER }}
```

Comment:
Will this path work now that it's value is `platformsh/template-builder` not `template-builder`?

Reply:
and this is why we do reviews! :D 
repository should have been replaced with the env, not path. i'll fix that.

Reply:
we _do_ reference that path though in another location so it wouldn't hurt to make it a variable.